src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -55,7 +109,7 @@ const CpsOnwardJourney = ({
   const CpsOnwardJourneyWrapper = ({ children }) =>
     enableGridWrapper ? (
       <GridWrapper {...a11yAttributes}>
```

Comment:
I noticed we are passing `{...a11yAttributes}` here and on line 115. Could we not just pass this prop in once, maybe pass it to `CpsOnwardJourneyWrapper` on line 129.

Reply:
:bulb: hmm @thekp that'd still mean we'd have to pass down `...restProps` and `children` down to `CpsOnwardJourneyWrapper` right ?

Reply:
oh yes true, sorry for some reason I saw this function like the one on 124-126 but it's not 🙃 you are right, keep it how it is 👍